webshopping/lolshopping/models.py

```diff
@@ -90,6 +90,17 @@ def imageURL(self):
 class Customer(models.Model):
     user = models.OneToOneField(Account, on_delete=models.CASCADE, null=True, blank=True)
     name = models.CharField(max_length=200, null=True)
+    # Personal information
+    full_name = models.CharField(max_length=100, null=True)
+    email = models.CharField(max_length=100, null=True)
+    phone_number = models.CharField(max_length=12, null=True)
+    #shipping address
+    address_line_1 = models.CharField(max_length=100, null=True)
+    address_line_2 = models.CharField(max_length=100, null=True)
+    city = models.CharField(max_length=50, null=True)
+    state = models.CharField(max_length=50, null=True)
+    postal_code = models.CharField(max_length=20, null=True)
```

Comment:
Any reasons why postal code and phone number are strings and not integers?

Reply:
my reason was some people put their numbers like "+1 702-555-1212" also the same thing with postal code, some include letters and hyphens. but I changed it decimal fields.

Reply:
It wouldn't matter since you could have it where if the user inputs a character that's not a number, it would be ignored and not be shown in the field.

Reply:
That's a bad way to handle data since that means there could be many different combinations of ways for users to input their  data. It could be anything within (123) 456-789, [123] 456_789, 123 (456) 789, you get get the idea. And if you want to actually retrieve this information in a useful way, then you'd have to process those information into a readable format. The best way to deal with this is to store it in the correct way in the first place. Now you changed it to decimal, which means user can be stupid and enter something like 123456.789. Which also would not make sense.

Reply:
I tried with integer field, it won't let me set the size of the number. so I decide to use the decimal field instead but modified it where it has to be a whole number.